### DISCUSSION TITLE
DeathPenalty defaulting to All

### DISCUSSION THREAD

**Discussion by crulorys** [thijsvanloef/palworld-server-docker#47] [Q&A] [answered]
Got the docker image up and running quite easily, however the DeathPenalty setting in DefaultPalworldSettings.ini is defaulted to All. I would prefer to have it set to 2. Would it be possible to make the DeathPenalty an environment setting when building the container using docker run? If not is there a way to modify that setting when the docker is already built?

Thanks in advance, this docker and the guide to get it started was super helpful and simple, We're running it on an Ubuntu 22 server, if that helps at all. Thanks again.

**Comment by thijsvanloef** [comment, marked as answer]
A new version has been released allowing you to set the Death Penalty using environment variables https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#editing-server-settings

The following options are allowed: None, Item, ItemAndEquipment, All
